apps/prairielearn/src/lib/load.ts

```diff
@@ -35,6 +35,7 @@ class LoadEstimator {
     this.warnOnOldJobs = warnOnOldJobs == null ? true : warnOnOldJobs;
     this.timeoutID = null;
     this.active = true;
+    // Start the load reporting loop
     this._reportLoad();
```

Comment:
This is not awaited, and given it was not awaited before as well I imagine it is on purpose. It might be good to add a comment to clarify that this is on purpose.

Reply:
Done in [896f595](https://github.com/PrairieLearn/PrairieLearn/pull/12437/commits/896f595a8300398a754c7427781bb51eba87d9c6)